CMakeLists.txt

```
@@ -120,6 +120,9 @@ if (OPENASSETIO_ENABLE_PYTHON)
         ""
         CACHE STRING
         "Override default Python module install directory, relative to CMAKE_INSTALL_PREFIX")
+
+    option(OPENASSETIO_ENABLE_PYTHON_INSTALL_DIST_INFO
+        "Create a dist-info metadata directory alongside Python installation" ON)
```

[Consider][Minor]
Just trying to be helpful to folks who might not understand the point of this.
```suggestion
    option(OPENASSETIO_ENABLE_PYTHON_INSTALL_DIST_INFO
        "Create a dist-info metadata directory alongside Python installation. Allows pip to understand that OpenAssetIO is installed." ON)
```

---

Augmented the description in 198c7f25